scripts/cc-cli/src/commands/bond.ts

```diff
@@ -59,7 +59,7 @@ async function bondAction(options: OptionValues) {
   console.log("Creating bond transaction...");
   console.log("Controller address:", options.controller);
   console.log("Reward destination:", rewardDestination);
-  console.log("Amount:", parseInt(options.amount, 10));
+  console.log("Amount:", parseFloat(options.amount));
```

Comment:
```suggestion
  console.log("Amount:", amount);
```

b/c it's already parsed